### PR TITLE
fix: adds more missing types

### PR DESCRIPTION
**Pull request by floryst**
### Context

This PR adds more missing types to vtk.js

### Results

Better type coverage

### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### CODE REVIEW

File: Sources/Rendering/OpenGL/HardwareSelector/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+   *
+   * @param inDispPos The input diplay position.
+   * @param maxDistance The max distance to consider from the input position.
+   * @param outDispPos The output display position.
+   */
+  getPixelInformation(
+    inDispPos: Vector2,
+    maxDistance: number,
+    outDispPos: Vector2
+  ): PixelInformation | null;
```
Comment (daker) [inline Sources/Rendering/OpenGL/HardwareSelector/index.d.ts:178]: `Nullable<PixelInformation>`

**Review by daker — CHANGES_REQUESTED** [review]

**Comment by floryst** [comment]
Updated to use `Nullable`.

**Review by daker — APPROVED** [review]
LGTM

File: Sources/Rendering/Core/HardwareSelector/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import { vtkObject } from '../../../interfaces';
+import vtkRenderer from '../../../Rendering/Core/Renderer';
+
+export interface vtkHardwareSelector extends vtkObject {
+  /**
+   * Get the picking source data.
+   *
+   * @param {vtkRenderer} renderer
+   * @param {number} fx1 bottom left x coord
+   * @param {number} fy1 bottom right x coord
```
Comment (daker) [inline Sources/Rendering/Core/HardwareSelector/index.d.ts:12]: fx2 instead?

File: Sources/Rendering/Core/HardwareSelector/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  /**
+   * Generates a selection.
+   *
+   * @param {vtkRenderer} renderer
+   * @param {number} fx1 bottom left x coord
+   * @param {number} fy1 bottom right x coord
+   * @param {number} fx2 bottom left y coord
+   * @param {number} fy2 bottom right y coord
+   */
```
Comment (daker) [inline Sources/Rendering/Core/HardwareSelector/index.d.ts:32]: Same here

**Review by daker — CHANGES_REQUESTED** [review]

File: Sources/Rendering/Core/HardwareSelector/index.d.ts (hunk 2 above)
Comment (floryst) [reply Sources/Rendering/Core/HardwareSelector/index.d.ts]: I've fixed these comments.

**Review by daker — APPROVED** [review]
LGTM

**Pull request merged**